content/en/armory-enterprise/quick-spin-instructions/_index.md

```diff
@@ -39,8 +48,16 @@ The provided application deploys a basic NGINX instance and sets up a minimal de
 1. Click *Save Changes*.
 1. Select *Pipelines* for the left navigation menu and click *Start Manual Execution* for the `prepare-quick-spin-environment` pipeline.
 
-{{< alert color="success" >}}Your pipeline is deployed. When it completes you can try out the `basic-deployment` pipeline. Use the `teardown-qick-spin environment` pipeline to remove the a deployment.{{< /alert >}}
+{{< alert color="success" >}}Your pipeline is deployed. When it completes you can try out the `basic-deployment` pipeline. 
+{{< /alert >}}
+
+- To learn more about each pipeline, see the [Quick Spin Pipelines](/armory-enterprise/quickspin/pipelines/) page.
```

Comment:
```suggestion
- To learn more about each pipeline, see the [Quick Spin Pipelines]({{< ref "/armory-enterprise/quickspin/pipelines/" >}}) page.
```
Please no hard-coded URLs ;-p. and also this link produces a 404 because it has /quickspin/